app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count = 0
+    without_mask = 0
+    without_vaccine = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
```

Comment:
Consider catching specific exceptions like `NotVaccinatedError` and `OutdatedVaccineError` separately if different actions are needed for each. This will provide more granular control over exception handling.